addon/adapters/osf-adapter.js

```diff
@@ -125,7 +125,7 @@ export default DS.JSONAPIAdapter.extend(HasManyQuery.RESTAdapterMixin, GenericDa
             var relatedType = singularize(snapshot.record[relationship].meta().type);
             res.data.forEach(item => {
                 var record = store.push(store.normalize(relatedType, item));
-                snapshot.record.resolveRelationship(relationship).addCanonicalRecord(record._internalModel.createSnapshot());
+                snapshot.record.resolveRelationship(relationship).addCanonicalRecord(record);
```

Comment:
I like that we're not relying on a chain of multiple private methods- was this found to be simply unnecessary? (I don't see a larger refactoring that changes the type of data being passed in)